tools/image-tag

```diff
@@ -2,14 +2,20 @@
 
 set -o errexit
 set -o nounset
+set -o pipefail
 
-SHA="$(git rev-parse --short HEAD)"
+WIP=$(git diff --quiet || echo '-WIP')
```

Comment:
Just to double check, this will create images for every commit to the branch? Not sure if we need that, but if it helps standardise this script across all our projects let's do it!

Reply:
I think `-WIP` will only be added if you run it locally. Drone will only run on new commits to the `main` and `r??` branches or new tags.

@jvrplmlmn am I correct that this is intended to add `-WIP` to your version if you build an image locally while you have changes?

Reply:
@kvrhdn you are correct 👍